yashsachdev.HabitTracker/HabitEnrollRepo.cs

```diff
@@ -0,0 +1,133 @@
+                command.Connection = cnn;
+                command.CommandText = "SELECT 1 FROM Habit_Enroll INNER JOIN Habit ON Habit_Enroll.Habit_Id = Habit.Habit_Id WHERE Habit_Enroll.User_Id = @userId AND Habit.Habit_Name = @habitname";
+                command.Parameters.AddWithValue("@userId", userId);
+                command.Parameters.AddWithValue("@habitname", habitName);
+                var reader = command.ExecuteReader();
+                return reader.Read();
+            }
+        }
+    }
+    public void DisplayUserHabit(string email)
+    {
+        UserRepo userRepo = new UserRepo();
+        HabitRepo habitRepo = new HabitRepo();
+        using (SqliteConnection cnn = new SqliteConnection(DatabaseClass.connectionString))
+        {
+            cnn.Open();
+            using (SqliteCommand command = new SqliteCommand())
+            {
+                command.Connection = cnn;
+                var user_ID = userRepo.GetIdFromEmail(email);
+                command.CommandText = "SELECT * FROM Habit_Enroll WHERE User_Id = @userId";
+                command.Parameters.AddWithValue("@userId", user_ID);
+                using (var reader = command.ExecuteReader())
+                {
+                    while (reader.Read())
+                    {
+                        var habit_ID = reader.GetInt32(1);
+                        var startdate = reader.GetDateTime(2);
+                        var habitName = habitRepo.GetHabitName(habit_ID);
+                        Console.WriteLine($"Habit : {habitName} \t Start Date:{startdate} \t");
+                    }
+                }
+            }
+
+        }
+    }
+    public void UpdateUserHabit(string email, string habitName, string updatedunit, int Habit_Id)
+    {
+        try
+        {
+            HabitRepo habitRepo = new HabitRepo();
+            using (SqliteConnection cnn = new SqliteConnection(DatabaseClass.connectionString))
+            {
+                cnn.Open();
+                using (SqliteCommand command = new SqliteCommand())
+                {
+                    command.Connection = cnn;
+                    habitRepo.UpdateHabitTable(habitName, Habit_Id, updatedunit, cnn);
+                }
+            }
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine("Error in UpdateUserHabit()");
+            Console.WriteLine(ex.Message);
+        }
+    }
+    internal void GenerateReport(int user_ID)
+    {
+        using (SqliteConnection cnn = new SqliteConnection(DatabaseClass.connectionString))
+        {
+            cnn.Open();
+            using (SqliteCommand command = new SqliteCommand())
+            {
+                command.Connection = cnn;
+                command.CommandText = "SELECT user.Name,habit.Habit_Name,habit.Unit,habitenroll.Date FROM Habit_Enroll habitenroll JOIN Habit habit ON habitenroll.Habit_Id = habit.Habit_Id JOIN User user ON habitenroll.User_Id =user.User_Id WHERE user.User_Id =@userid";
```

Comment:
use verbatim strings to achieve something like this: 

![image](https://user-images.githubusercontent.com/108711476/220471467-4f9f2151-edff-433b-b64a-b6616c5da308.png)